google-cloud-spanner-hibernate-testing/src/test/java/org/hibernate/userguide/sql/SQLTest.java

```diff
@@ -402,27 +409,6 @@ public void test_sql_hibernate_entity_associations_query_many_to_one_join_result
 		});
 	}
 
-	@Test
-	@RequiresDialect(H2Dialect.class)
-	@RequiresDialect(Oracle8iDialect.class)
-	@RequiresDialect(PostgreSQL82Dialect.class)
-	public void test_sql_jpa_entity_associations_query_one_to_many_join_example() {
```

Comment:
Why would you need to delete this, if you it doesn't relate to the Spanner dialect anyway?

Reply:
I think it's ok to delete this because this is just gonna be dead code as it is only run when those specific dialects are loaded.